src/main/js/main_html5.js

```diff
@@ -872,6 +872,10 @@ require("../../../html5-common/js/utils/environment.js");
      * @method OoyalaVideoWrapper#raiseWaitingEvent
      */
     var raiseWaitingEvent = _.bind(function() {
+      // WAITING event is not raised if no video is assigned yet
+      if (_video.currentSrc === "" || _video.currentSrc === null) { 
```

Comment:
I recommend to use if (_.isEmpty(_video.currentSrc)) { - it covers all the cases